modules/http/src/main/java/org/mule/module/http/internal/domain/request/HttpRequestBuilder.java

```diff
@@ -39,9 +45,22 @@ public HttpRequestBuilder setHeaders(ParameterMap headers)
         return this;
     }
 
-    public HttpRequestBuilder addHeader(String headerName, String headerValue)
+    public HttpRequestBuilder addHeader(String headerName, Object headerValue)
```

Comment:
I think this should maintain it's signature and the outbound properties should be analysed instead.